core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -93,12 +93,14 @@ public static void initializeNatives(Path tmpFile, boolean mlockAll, boolean sec
                 throw new RuntimeException("don't run elasticsearch as root.");
             }
         }
-        
+
         // enable secure computing mode
         if (seccomp) {
             Natives.trySeccomp(tmpFile);
+        } else {
+            logger.warn("Not enabling seccomp because it was explicitly disabled");
         }
```

Comment:
Yeah - it looks like some kind of remote executor helper is required for this. I feel like it'd be easier to yank some of the code we already use to start the bwc nodes into a java process started into pre-integration-test than to write it in perl and fight with windows.